DependencyInjection/DoctrineExtension.php

```diff
@@ -73,6 +73,20 @@ public function load(array $configs, ContainerBuilder $container)
 
         $this->adapter->loadServicesConfiguration($container);
 
+        // Use a prototype when the ORM is available.
+        // This avoids listing all ORM commands by hand.
+        if (class_exists(Version::class)) {
```

Comment:
This type of check should be avoided since Version class is being removed in 3.0, see https://github.com/doctrine/doctrine2/pull/6653.
Probably good enough for now (already used in registerCommands()), but should be handled differently in future.